docs/development/module/tests.md

```diff
@@ -116,16 +128,6 @@ fn my_runtime_test() {
 }
 ```
 
-## Next Steps
-
-### Learn More
-
-TODO
-
-### Examples
-
-TODO
-
-### References
+## References and Examples
 
-TODO
+The [testing](https://substrate.dev/recipes/base/testing/index.html) chapter of the [Substrate Recipes](https://github.com/substrate-developer-hub/recipes/) compliments the samples shown above, and the [kitchen](https://github.com/substrate-developer-hub/recipes/tree/master/kitchen) provides an environment to run the tests, change the logic, and tinker with the code.
```

Comment:
Testing link is broken

Reply:
It won't work until the PR in recipes #87 is merged.